### ISSUE TITLE
[remote ssh] connectionToken always consist of '1' and 'a'

### ISSUE THREAD

**Issue by ladyrick** [microsoft/vscode-remote-release#7560]
- VSCode Version: 1.73.1
- Local OS Version: macos 13.0
- Remote OS Version: Debian GNU/Linux 9.13 (stretch)
- Remote Extension/Connection Type: SSH
- Logs:
```
[15:22:35.586] Received install output: 
SSH_AUTH_SOCK====
DISPLAY====
webUiAccessToken====
listeningOn==13773==
osReleaseId==debian==
arch==x86_64==
tmpDir==/run/user/0==
platform==linux==
unpackResult====
didLocalDownload==0==
downloadTime====
installTime====
extInstallTime====
serverStartTime====
connectionToken==aa1a11aa-aaa1-1a11-aa11-1a1aaa111a11==
```

Steps to Reproduce:

1. connect to remote ssh vscode
2. open "output" -> "Remote - SSH"
3. see the log

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: No

As you can see, the connectionToken is `aa1a11aa-aaa1-1a11-aa11-1a1aaa111a11`. Even I try to restart vscode, the token just changes to another combination of '1' and 'a'.

**Comment by eleanorjboyd** [comment]
Hello! To clarify are you able connect successfully? Additionally are these the full logs?

**Comment by ladyrick** [comment]
> 

I connected successfully. Here is the full log:
```
[01:37:48.954] Log Level: 2
[01:37:48.955] remote-ssh@0.92.0
[01:37:48.955] darwin x64
[01:37:48.961] SSH Resolver called for "ssh-remote+vm", attempt 1
[01:37:48.962] "remote.SSH.useLocalServer": true
[01:37:48.962] "remote.SSH.path": undefined
[01:37:48.962] "remote.SSH.configFile": ~/.ssh/config_vscode
[01:37:48.962] "remote.SSH.useFlock": true
[01:37:48.962] "remote.SSH.lockfilesInTmp": false
[01:37:48.963] "remote.SSH.localServerDownload": auto
[01:37:48.963] "remote.SSH.remoteServerListenOnSocket": false
[01:37:48.963] "remote.SSH.showLoginTerminal": false
[01:37:48.963] "remote.SSH.defaultExtensions": []
[01:37:48.963] "remote.SSH.loglevel": 2
[01:37:48.963] "remote.SSH.enableDynamicForwarding": true
[01:37:48.963] "remote.SSH.enableRemoteCommand": false
[01:37:48.963] "remote.SSH.serverPickPortsFromRange": {}
[01:37:48.964] "remote.SSH.serverInstallPath": {}
[01:37:48.966] SSH Resolver called for host: vm
[01:37:48.966] Setting up SSH remote "vm"
[01:37:48.969] Acquiring local install lock: /var/folders/bk/g7ckffn56ll7g3031lgd7kth0000gp/T/vscode-remote-ssh-4049c22c-install.lock
[01:37:48.971] Looking for existing server data file at /Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-4049c22c-6261075646f055b99068d3688932416f2346dd3b-0.92.0/data.json
[01:37:48.971] Using commit id "6261075646f055b99068d3688932416f2346dd3b" and quality "stable" for server
[01:37:48.974] Install and start server if needed
[01:37:48.977] PATH: /Users/username/.pyenv/shims:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/puppetlabs/bin
[01:37:48.977] Checking ssh with "ssh -V"
[01:37:48.984] > OpenSSH_9.0p1, LibreSSL 3.3.6

[01:37:48.986] Using SSH config file "/Users/username/.ssh/config_vscode"
[01:37:48.987] askpass server listening on /var/folders/bk/g7ckffn56ll7g3031lgd7kth0000gp/T/vscode-ssh-askpass-c08839bea988f54bbd0dc5c6f53af39c2db8c2ae.sock
[01:37:48.987] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/bk/g7ckffn56ll7g3031lgd7kth0000gp/T/vscode-ssh-askpass-7f4098306f365978ce0ef546cee3a7a70df29146.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","51080","-o","ConnectTimeout=15","-F","/Users/username/.ssh/config_vscode","vm"],"serverDataFolderName":".vscode-server","dataFilePath":"/Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-4049c22c-6261075646f055b99068d3688932416f2346dd3b-0.92.0/data.json"}
[01:37:48.987] Local server env: {"SSH_AUTH_SOCK":"/private/tmp/com.apple.launchd.x9zuYRUpZA/Listeners","SHELL":"/bin/zsh","DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.92.0/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"--ms-enable-electron-run-as-node","VSCODE_SSH_ASKPASS_MAIN":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.92.0/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/bk/g7ckffn56ll7g3031lgd7kth0000gp/T/vscode-ssh-askpass-c08839bea988f54bbd0dc5c6f53af39c2db8c2ae.sock"}
[01:37:48.988] Spawned 65887
[01:37:49.148] > local-server-1> Spawned ssh, pid=65891
[01:37:49.185] stderr> OpenSSH_9.0p1, LibreSSL 3.3.6
[01:37:49.363] stderr> debug1: Server host key: ssh-ed25519 SHA256:Oec+DgiFE7VfDq0cM+mc771Dutb8M/tcIc2NBj1ZHzU
[01:37:49.543] stderr> 
[01:37:49.543] stderr> 
[01:37:49.560] stderr> unknown mech-code 0 for mech 1 2 752 43 14 2
[01:37:49.560] stderr> 
[01:37:49.561] stderr> 
[01:37:49.561] stderr> unknown mech-code 0 for mech 1 3 6 1 5 5 14
[01:37:49.561] stderr> 
[01:37:49.561] stderr> 
[01:37:49.628] stderr> unknown mech-code 2 for mech 1 3 6 1 4 1 311 2 2 10
[01:37:49.628] stderr> 
[01:37:49.628] stderr> 
[01:37:49.628] stderr> unknown mech-code 0 for mech 1 3 5 1 5 2 7
[01:37:49.628] stderr> 
[01:37:49.628] stderr> 
[01:37:49.628] stderr> unknown mech-code 0 for mech 1 3 6 1 5 2 5
[01:37:49.628] stderr> 
[01:37:49.628] stderr> 
[01:37:49.660] stderr> Authenticated to xx.xx.xx.xx ([xx.xx.xx.xx]:22) using "publickey".
[01:37:49.747] > Welcome to Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-52-generic x86_64)
> 
>  * Documentation:  https://help.ubuntu.com
>  * Management:     https://landscape.canonical.com
>  * Support:        https://ubuntu.com/advantage
[01:37:49.756] > ready: 4c03723552b8
[01:37:49.768] > Linux 5.15.0-52-generic #58-Ubuntu SMP Thu Oct 13 08:03:55 UTC 2022
[01:37:49.768] Platform: linux
[01:37:49.778] > /bin/bash
[01:37:49.779] Shell: bash
[01:37:49.779] Creating bash subshell inside "bash"
[01:37:49.791] > bash version:  5.1.16(1)-release
[01:37:49.808] > 4c03723552b8: running
[01:37:49.827] > Acquiring lock on /home/username/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/vscode-remote-lock.username.6261075646f055b99068d3688932416f2346dd3b
[01:37:49.830] > Found existing installation at /home/username/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b...
> Checking /home/username/.vscode-server/.6261075646f055b99068d3688932416f2346dd3b.log and /home/username/.vscode-server/.6261075646f055b99068d3688932416f2346dd3b.pid for a running server
[01:37:49.831] > Looking for server with pid: 2430398
[01:37:49.852] > Found running server...
[01:37:49.853] >  
> *
> * Visual Studio Code Server
> *
> * By using the software, you agree to
> * the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
> * the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
> *
>  
[01:37:49.860] > Checking server status on port 42269 with wget
[01:37:49.869] > 4c03723552b8: start
> SSH_AUTH_SOCK====
> DISPLAY====
> webUiAccessToken====
> listeningOn==42269==
> osReleaseId==ubuntu==
> arch==x86_64==
> tmpDir==/run/user/1000==
> platform==linux==
> unpackResult====
> didLocalDownload==0==
> downloadTime====
> installTime====
> extInstallTime====
> serverStartTime====
> connectionToken==aaaa11a1-aa11-11a1-a1a1-aaa1111a1a1a==
> 4c03723552b8: end
[01:37:49.870] Received install output: 
SSH_AUTH_SOCK====
DISPLAY====
webUiAccessToken====
listeningOn==42269==
osReleaseId==ubuntu==
arch==x86_64==
tmpDir==/run/user/1000==
platform==linux==
unpackResult====
didLocalDownload==0==
downloadTime====
installTime====
extInstallTime====
serverStartTime====
connectionToken==aaaa11a1-aa11-11a1-a1a1-aaa1111a1a1a==

[01:37:49.870] Remote server is listening on 42269
[01:37:49.870] Parsed server configuration: {"serverConfiguration":{"remoteListeningOn":{"port":42269},"osReleaseId":"ubuntu","arch":"x86_64","webUiAccessToken":"","sshAuthSock":"","display":"","tmpDir":"/run/user/1000","platform":"linux","connectionToken":"aaaa11a1-aa11-11a1-a1a1-aaa1111a1a1a"},"installUnpackCode":""}
[01:37:49.872] Persisting server connection details to /Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-4049c22c-6261075646f055b99068d3688932416f2346dd3b-0.92.0/data.json
[01:37:49.876] Starting forwarding server. localPort 51082 -> socksPort 51080 -> remotePort 42269
[01:37:49.877] Forwarding server listening on 51082
[01:37:49.877] Waiting for ssh tunnel to be ready
[01:37:49.879] Tunneled 42269 to local port 51082
[01:37:49.879] Resolved "ssh-remote+vm" to "127.0.0.1:51082"
[01:37:49.881] [Forwarding server 51082] Got connection 0
[01:37:49.890] ------




[01:37:49.906] [Forwarding server 51082] Got connection 1
[01:37:49.943] [Forwarding server 51082] Got connection 2
[01:37:50.785] [Forwarding server 51082] Got connection 3
```

**Comment by eleanorjboyd** [comment]
Hello! Yes this is desired behavior. We censor the connnectionToken in the logs which is why it comes out as a string of 1s and a's.
